src/public/curriculum/example-curriculum/example-variables.json

```diff
@@ -32,7 +32,7 @@
         "numFormat": ".2~f"
       },
       "text": {
-        "tools": ["bold", "subscript", "superscript", "m2s-variables"]
+        "tools": ["bold", "subscript", "superscript", "m2s-variables", "new-variable", "insert-variable", "edit-variable"]
```

Comment:
I added these everywhere (3 places) I found "m2s-variables" in json files, but I'm not actually sure they belong in all of them.